src/components/v5/shared/Filter/FilterButton.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { FC } from 'react';
+import { useIntl } from 'react-intl';
+import clsx from 'clsx';
+
+import styles from './FilterButton.module.css';
+import Icon from '~shared/Icon';
+import { FilterButtonProps } from './types';
+
+const displayName = 'v5.shared.Filter.FilterButton';
+
+const FilterButton: FC<FilterButtonProps> = ({
+  isOpen,
+  selectedFilterNumber,
+  ref,
+  onClick,
+}) => {
+  const { formatMessage } = useIntl();
+
+  return (
+    <button
+      type="button"
+      aria-label={formatMessage({ id: 'filter.button' })}
+      className={clsx(styles.filterButton, {
+        'border border-blue-400 text-blue-400': isOpen,
+      })}
+      ref={ref}
+      onClick={onClick}
+    >
+      <Icon name="funnel-simple" appearance={{ size: 'tiny' }} />
+      {formatMessage({ id: 'filter' })}
+
+      {!!selectedFilterNumber && (
+        <span className="bg-blue-100 p-1 rounded-sm text-blue-400 font-bold text-xs h-[0.813rem] flex items-center">
```

Comment:
This value i't correct: h-[0.813rem] - please, use full values

Reply:
Please add to global.css and main.css a new typography text style with text-bold and text-xs and use it here